lib/xgif/xgif_test.go

```diff
@@ -0,0 +1,35 @@
+package xgif
+
+import (
+	_ "embed"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//go:embed test_input1.png
+var test_input1 []byte
+
+//go:embed test_input2.png
+var test_input2 []byte
+
+//go:embed test_output.gif
+var test_output []byte
+
+func TestPngToGif(t *testing.T) {
+	boards := [][]byte{test_input1, test_input2}
+	interval := 1_000
+	gifBytes, err := AnimatePNGs(boards, interval)
+	assert.NoError(t, err)
+
+	// use this to update the test output
+	if false {
```

Comment:
shouldn't this just skip in ci only?

Reply:
my intent here is just to leave the code ready so that, if someone needs to view the generated output, just changes to `true`